docs/source/manual/openfpga_shell/openfpga_commands/setup_commands.rst

```diff
@@ -460,6 +460,10 @@ pcf2place
   .. option:: --no_time_stamp
 
     Do not print time stamp in output files
+  
+  .. option:: --reduce_error_to_warning
+
+    Reduce error to warning while reading commands in pcf file
```

Comment:
Add a warning message to alert users that they should be very cautious when enable this option. 

See an example here: https://docs.verilogtorouting.org/en/latest/vpr/command_line_usage/#cmdoption-vpr-strict_checks